recipes/catch2/3.x.x/conanfile.py

```diff
@@ -7,7 +7,7 @@
 import os
 import textwrap
 
-required_conan_version = ">=1.53.0"
+required_conan_version = ">=2.1.0"
```

Comment:
```suggestion
required_conan_version = ">=1.53.0 <2 || >=2.1.0"
```
Perhaps it's possible to still allow Conan v1 support, but with the responsibility of the user to make sure that `--build` is used for incompatible cppstd versions.

Reply:
Thanks for the suggestion, that sounds sensible.